lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
+// ====================
+
+// home - layout
+.container .discussion-body {
```

Comment:
If the former is true of [my previous comment](https://github.com/edx/edx-platform/pull/9666/files#r39049227), this selector seems pretty generic and not tied to that "home" view. Is there a body-based class or class added to a container that we can be more specific from?